_rules/html-lang-valid-bf051a.md

```diff
@@ -160,20 +168,4 @@ The rule applies to `html` element and hence usage of `xml:lang` attribute in `s
 
 ```svg
 <svg xmlns="http://www.w3.org/2000/svg" xml:lang="fr"></svg>
-```
-
-#### Inapplicable Example 3
```

Comment:
Why have inapplicable examples 3 and 4 been removed?
(and you should even add examples with whitespace attributes)

Reply:
@Jym77 these test cases don't satisfy the accessibility requirement. In order for the mapping to work, we can only have passed / inapplicable test cases that also satisfy the success criterion.

Reply:
OK. I think I see the point.

I'm still a bit uneasy with that because it means that the test cases don't match the accessibility requirement **mapping** of the rule. Namely, in that case, we say that if the rule passes or is inapplicable, then further testing is needed, but we have no example that show that further testing is actually needed.

Reply:
@Jym77 your concern about whether further testing is needed may be something that needs a bigger discussion & may apply to a larger subset of rules. Can you open an issue to enable discussion on this?